Classes/Level1Scene.cpp

```diff
@@ -50,6 +50,43 @@ bool Level1Scene::init()
         this->addChild(background, 0);
     }
 
+    //create map
+    float delta = 140;
+    float x = origin.x + 70 + delta;
+    float y = origin.y + visibleSize.height - 70 - delta;
+    float SIZE = 140.0;
+    ui::Button* grid[7][12]={};
+    std::string type[7][12]={};
```

Comment:
Construct a map from `core::Map` and draw the ui according to it?

Reply:
This is just a test of the pictures showing the map to see if these pictures need to be changed.